env/values-prod.yaml

```diff
@@ -203,12 +191,15 @@ resources: {}
   #   memory: 128Mi
 
 autoscaling:
-  enabled: false
-  minReplicas: 1
-  maxReplicas: 100
+  enabled: true
+  minReplicas: 3
+  maxReplicas: 10
   targetCPUUtilizationPercentage: 80
   # targetMemoryUtilizationPercentage: 80
 
+pdb:
+  maxUnavailable: 1
+
```

Comment:
these blocks need to be under each of the service (i.e .Values.client.autoscaling/ .Values.client.pdb) and make sure each service has it's own PDB and HPA file (client-hpa, server-hpa, etc)